cmd/grpcui/grpcui.go

```diff
@@ -351,6 +361,18 @@ func main() {
 	flags.Usage = usage
 	flags.Parse(os.Args[1:])
 
+	gRPCOptions := []string{}
+	for _, flagName := range grpcCurlFlags {
+		f := flags.Lookup(flagName)
+		if f.Value.String() != f.DefValue {
+			if f.Value.String() == "true" {
```

Comment:
```suggestion
			if getter, ok := f.Value.(flag.Getter); ok && getter.Get() == true {
```
Otherwise you could break on a string value that happens to be the string true.